tests/kernel/common/src/clock.c

```diff
@@ -157,6 +189,42 @@ static void init_data_count(void)
  *
  */
 
+/**
+ * @brief Test millisecond time duration
+ *
+ * @details
+ * Test Objective:
+ * - This test verify kernel support be provided time duration in millisecond
+ *
+ * Testing techniques:
+ * - Interface testing
+ *
+ * Prerequisite Conditions:
+ * - N/A
+ *
+ * Input Specifications:
+ * - N/A
+ *
+ * Test Procedure:
+ * -# Initialize a timer.
+ * -# Provide time duration in millisecond.
+ * -# Start timer, and check the duration time whether correct.
+ * -# Stop the timer.
+ *
```

Comment:
The test procedure has to be written to describe the details. For example:
1. Start a timer with 100ms.
2. Busy wait 80ms.
3, Check if the timer has expired.